### PR TITLE
Introduces ChromeHeadless, FirefoxHeadless and enzyme@3.0.0

### PR DESCRIPTION
**Pull request by KaiVolland**
This replaces PhantomJS with [ChromeHeadless](https://github.com/karma-runner/karma-chrome-launcher/tree/v2.2.0#available-browsers) an [FirefoxHeadless](https://github.com/karma-runner/karma-firefox-launcher/issues/76#issuecomment-328349047)  and introduces version 3.0.0 of enzyme. :heart: 

Currently failing (outcommented) tests:
LayerTree
- onCheck
  - sets the correct visibility to the layer from the checked state
- event handling
  - sets checked state corresponding to layer.setVisible

As these tests affect the LayerTree / LayerTreeNode which we want to refactor anyway i'll think it's not that important...

### CODE REVIEW

**Comment by marcjansen** [comment]
♥️♥️♥️ will look into this tomorrow

**Comment by KaiVolland** [comment]
@terrestris/devs plz review

File: src/Button/SimpleButton/SimpleButton.spec.jsx [hunk 1]
```diff
@@ -56,9 +56,7 @@ describe('<SimpleButton />', () => {
     const wrapper = TestUtil.mountComponent(SimpleButton);
     const onClick = sinon.spy();
 
-    wrapper.setProps({
-      onClick: onClick
-    });
+    wrapper.setProps({onClick});
```
Comment (dnlkoch) [inline src/Button/SimpleButton/SimpleButton.spec.jsx]: Initial props may be moved to `mountComponent` completely.

File: src/Panel/Titlebar/Titlebar.spec.jsx [hunk 2]
```diff
@@ -10,41 +10,40 @@ import {
...
+    });
     expect(wrapper).not.to.be(undefined);
   });
 
   it('adds a passed className', () => {
-    const wrapper = setup({className: 'podolski'});
-    expect(wrapper.node.props.className).to.contain('podolski');
+    const wrapper = TestUtil.mountComponent(Titlebar, {
+      parent: document.body,
+      className: 'podolski'
```
Comment (dnlkoch) [inline src/Panel/Titlebar/Titlebar.spec.jsx]: :-1:

File: src/Util/TestUtil.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (dnlkoch) [inline src/Util/TestUtil.js]: Please move the global `Enzyme` configuration to an own file, e.g. `globals.js`.

**Comment by dnlkoch** [comment]
Very, very nice! I've added some minor remarks only.

File: src/Util/TestUtil.js (hunk 3 above)
Comment (marcjansen) [reply src/Util/TestUtil.js]: 👍

**Pull request merged**